credits.md

```diff
@@ -5,176 +5,39 @@ summary: Learn about the list and roles of contributors to the TiDB community.
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[ekalinin](https://github.com/ekalinin)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[erjiaqing](https://github.com/erjiaqing)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[hey-kong](https://github.com/hey-kong)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[jacklightChen](https://github.com/jacklightChen)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[k-ye](https://github.com/k-ye)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[pingyu](https://github.com/pingyu)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[Rustin-Liu](https://github.com/Rustin-Liu)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[spongedu](https://github.com/spongedu)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[TennyZhuang](https://github.com/TennyZhuang)|
-|[execution](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-exec)|[xiekeyi98](https://github.com/xiekeyi98)|
-|[ddl](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl)|[reafans](https://github.com/reafans)|
-|[ddl](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl)|[Rustin-Liu](https://github.com/Rustin-Liu)|
-|[ddl](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-ddl)|[spongedu](https://github.com/spongedu)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[Deardrops](https://github.com/Deardrops)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[foreyes](https://github.com/foreyes)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[lonng](https://github.com/lonng)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[SeaRise](https://github.com/SeaRise)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[tiancaiamao](https://github.com/tiancaiamao)|
-|[planner](https://github.com/pingcap/community/tree/master/special-interest-groups/sig-planner)|[wshwsh12](https://github.com/wshwsh12)|
+Contributors play an important role in the robust development of TiDB. We sincerely appreciate all contributors who submit code, write and translate documents for TiDB.
+
+## TiDB developers
+
+TiDB developers contribute to new feature development, performance optimization, and bugfixes. The following is the list of contributors to TiDB related repo:
```

Comment:
```suggestion
TiDB developers contribute to new feature development, performance improvement, and stability The following is the list of contributors in TiDB related repos:
```

Reply:
@TomShawn Did you remove the `.` on purpose?

Shouldn't that be `stability improvements` instead of `stability`? What about other bugfixes that are not related to stability?

Reply:
Sorry, that was not on purpose. This is a literal translation from the Chinese version: https://github.com/pingcap/docs-cn/pull/5901/files#diff-a3cfe79e912417e309dc5273eb5b3ef6b8c123f8f597286e1ea9ce12f7afbc6eR13. I have added bug fixes here. Later we will synchronize it back.